autogitops/autogitops.json

```diff
@@ -1,7 +1,39 @@
 {
-    "name": "loderunner",
     "namespace": "loderunner",
-    "imageName": "ghcr.io/retaildevcrews/ngsa-lr",
-    "imageTag": "beta",
-    "targets": ["dev-ngsa-asb-eastus","dev-ngsa-asb-westus2","pre-ngsa-asb-centralus","pre-ngsa-asb-eastus","pre-ngsa-asb-westus2"]
+    "targets": ["dev-ngsa-asb-eastus","dev-ngsa-asb-westus2","pre-ngsa-asb-centralus","pre-ngsa-asb-eastus","pre-ngsa-asb-westus2"],
+    "tenantid": "648dcb5a-de1e-48b2-af6b-fe6ef28d355c",
+    "resourceGroup": "rg-ngsa-asb-dev",
+    "keyvaultName": "kv-aks-3i2qzkkxofr7c",
+    "lrmi": {
```

Comment:
I dont remember if nested json is supported in autogitops.json. @bartr can you plz confirm?

Reply:
confirmed nested json not supported. fixed.